spec/controllers/playlists_controller_spec.rb

```diff
@@ -174,6 +174,53 @@
     end
   end
 
+  describe 'POST #replicate' do
+    before do
+      login_as :user
+    end
+    let(:new_attributes) do
+      { title: Faker::Lorem.word, visibility: Playlist::PUBLIC, comment: Faker::Lorem.sentence, user: user }
+    end
+    let(:playlist) { FactoryGirl.create(:playlist, new_attributes) }
+
+    context 'blank playlist' do
+      it 'replicate a blank playlist' do
+        post :replicate, format: 'json', old_playlist_id: playlist.id,
+          playlist: { 'title' => playlist.title, 'comment' => playlist.comment, 'visibility' => playlist.visibility }
+        expect(response.body).not_to be_empty
+        parsed_response = JSON.parse(response.body)
+
+        new_playlist = parsed_response['playlist']
```

Comment:
How about looking the playlist back up from the DB to ensure that it got persisted like in the test below?